TCAT/Views/DatepickerView.swift

```diff
@@ -56,14 +66,37 @@ class DatepickerView: UIView {
     
     // MARK: Segement Control
     
-    func segmentedControlValueChanged(segmentControl: UISegmentedControl) {
-        let leaveNow = 0
-        if segmentedControl.selectedSegmentIndex == leaveNow {
-            datepicker.date = Date()
-            segmentedControl.selectedSegmentIndex = prevSelectedIndex
+    func leaveNowSegmentedControlValueChanged(segmentControl: UISegmentedControl) {
```

Comment:
(Possibly) apply similar comments mentioned above (`let now = Date()` and `let leaveNow = 0`)